railties/lib/rails/commands/server/server_command.rb

```diff
@@ -99,7 +99,7 @@ class ServerCommand < Base # :nodoc:
 
       class_option :port, aliases: "-p", type: :numeric,
         desc: "Runs Rails on the specified port.", banner: :port, default: 3000
-      class_option :binding, aliases: "-b", type: :string, default: "localhost",
```

Comment:
I removed the default value. This is because the default value varies depending on the environment.